### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by laurent-mbuyu**
Registration pages

Thanks for reviewing and leaving your feedback

### CODE REVIEW

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]
Resolve merge c onflicts

File: src/components/registersteptwo.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import Registration from './registration';
+import Registerstepone from './registerstepone';
+import Registerstepthree from './registerstepthree';
+
+class Registersteptwo extends Component {
+
+    constructor(props)
+    {
+        super(props);
+        console.log(props);
```
Comment (KimberleighBeyl) [inline src/components/registersteptwo.js]: remove console.log

File: src/components/registration.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+class Registration extends Component {
+
+    constructor(props)
+    {
+        super(props);
+        this.state = {
+            step: 1,
+            name: null,
```
Comment (KimberleighBeyl) [inline src/components/registration.js]: no state for name, make a reducer

**Comment by laurent-mbuyu** [comment]
Requested changes have been addressed. Thanks.

File: src/components/registerstepone.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import Registersteptwo from './registersteptwo';
+import Registerstepthree from './registerstepthree';
+import MainChat from './mainChat';
+
+class Registerstepone extends Component {
+
+    constructor(props)
+    {
+        super(props);
+        console.log(props);
```
Comment (KimberleighBeyl) [inline src/components/registerstepone.js]: remove

File: src/components/registerstepone.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                </BrowserRouter>
+            </div>
+        )
+    }
+
+    saveAndContinue(e) {
+        e.preventDefault()
+    
+        // Get values via this.refs
+        var data = {
```
Comment (KimberleighBeyl) [inline src/components/registerstepone.js]: store in reducer

File: src/components/registerstepone.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+          name     : this.refs.name.value,
+          email    : this.refs.email.value,
+          password : this.refs.password.value,
+        }
+    
+        this.props.saveValues(data)
```
Comment (KimberleighBeyl) [inline src/components/registerstepone.js:51]: I don't see any reducers

**Review by KimberleighBeyl — CHANGES_REQUESTED** [review]
I don't see any reducers where data is stored

**Review by RashadAkoodieRabbit — APPROVED** [review]
Thank you

**Review by KimberleighBeyl — APPROVED** [review]
:+1:

**Pull request merged**